pkg/operator/ceph/csi/spec.go

```diff
@@ -890,16 +894,20 @@ func (r *ReconcileCSI) configureHolder(driver driverDetails, c ClusterDetail, tp
 			},
 		},
 	}
-	netNamespaceFilePath := generateNetNamespaceFilePath(CSIParam.KubeletDirPath, driver.fullName, c.cluster.Namespace)
-	if driver.name == RBDDriverShortName {
-		clusterConfigEntry.RBD.NetNamespaceFilePath = netNamespaceFilePath
-	}
-	if driver.name == CephFSDriverShortName {
-		clusterConfigEntry.CephFS.NetNamespaceFilePath = netNamespaceFilePath
-	}
-	if driver.name == NFSDriverShortName {
-		clusterConfigEntry.NFS.NetNamespaceFilePath = netNamespaceFilePath
+
+	if IsHolderEnabled() {
```

Comment:
i think this change is not required here, `configureHolder` called only when Holder needs to be configured

Reply:
It is called in `startDrivers` func and I don't see any condition around it.
https://github.com/rook/rook/blob/f85ba8808950384148aa7ca5fa0505beeeca9be8/pkg/operator/ceph/csi/spec.go#L408-L418

Reply:
https://github.com/rook/rook/blob/5ae25b60f3e3dad8a3e34549512b7849f9d7ace6/pkg/operator/ceph/csi/spec.go#L796-L805 check this one, its called only for the clusters which is having multus enabled clusters not for others. did i miss anything?

Reply:
I missed it,  removed the check.